conf/README.md

```diff
@@ -0,0 +1,84 @@
+# Config directory
+
+In this directory we will be storing all the configuration for clusters and
+OCSCI config files.
+
+During the execution we are via
```

Comment:
This sentence is a bit unclear. Change to something like:
During the execution, via [ocsci pytest plugin](../pytest_customization/ocscilib.py), we're loading all the...

Reply:
done